.circleci/config.yml

```diff
@@ -138,7 +138,12 @@ jobs:
       - run:
           name: Prepare the infrastructure
           command: e2e-commons/up.sh deploy native-to-erc
-      - run: 'echo "TODO - Run the e2e tests on top of the infrastructure created by previous step"'
+      - run:
+          name: Wait for the Oracle to start
+          command: 'sleep 120'
```

Comment:
I did try to have a more smart way to wait - waiting until `"Required block confirmations obtained"` record appears in the logs or similar.

I had trouble making that work, with the docker-in-docker situation and all, and eventually gave up - at least for now.

Reply:
Will it possible to get data from RedisDB to get information about observed blocks? By doing this (or even by watching if it changes) we could confirm that the oracle is up and running

Reply:
@rzadp :arrow_up:

Reply:
Yes, it most likely is possible, I will try to make it happen.

Reply:
Done - monitoring redis value of `native-erc-collected-signatures:lastProcessedBlock`.